two-data-centers-in-one-city-deployment.md

```diff
@@ -22,7 +22,7 @@ This section takes the example of a region where two availability zones AZ1 and
 
 The architecture of the cluster deployment is as follows:
 
-- The cluster has four replicas: two Voter replicas in AZ1, one Voter replica, and one Learner replica in AZ2. For the TiKV component, each rack has a proper label.
+- The cluster has six replicas: three Voter replicas in AZ1, two Voter replicas, and one Learner replica in AZ2. For the TiKV component, each rack has a proper label.
```

Comment:
I think this belongs to a different PR?

Reply:
It's also a feedback from AskDocs. So I updated it in the same PR. You can safely ignore it.